index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="ru">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles/styles.css" />
+    <script type="module" src="scripts/index.js"></script>
+    <title>Haker_news_JS</title>
+  </head>
+  <body>
+    <div class="container">
+      <header class="container__header">
+        <div class="logo">
+          <div class="logo__image"><img src="images/logo.gif" /></div>
```

Comment:
</img needs to have alt attribute

Reply:
Fixed.

Reply:
Update: the tag IMG is no longer in use. Logo is added via styles. In this case, it is not possible to add an attribute.